providers/default.rb

```diff
@@ -191,13 +191,26 @@ def chef_backup_dir
   "#{chef_install_dir}.upgrade"
 end
 
+def chef_locked_files_dir
+  "#{chef_install_dir}.locked_files"
+end
+
 # cleanup cruft from *prior* runs
 def cleanup
   if ::File.exist?(chef_backup_dir) # rubocop:disable Style/GuardClause
     converge_by("remove #{chef_backup_dir} from previous chef-client run") do
       FileUtils.rm_rf chef_backup_dir
     end
   end
+  if ::File.exist?(chef_locked_files_dir) # rubocop:disable Style/GuardClause
+    converge_by("remove #{chef_locked_files_dir} from previous chef-client run") do
+      begin
+        FileUtils.rm_r chef_locked_files_dir
+      rescue
+        Chef::Log.warn("Failed to remove locked files dir!")
```

Comment:
`"remove #{chef_locked_files_dir} from previous chef-client run"` this is more accurately a previous upgrade, right? should we say `"remove #{chef_locked_files_dir} from previous chef client upgrade"`?